serving/src/main/java/feast/serving/config/FeastProperties.java

```diff
@@ -61,6 +66,41 @@ public FeastProperties() {}
   /* Feast Core port to connect to. */
   @Positive private int coreGrpcPort;
 
+  private CoreAuthenticationProperties coreAuthentication;
+
+  public CoreAuthenticationProperties getCoreAuthentication() {
+    return coreAuthentication;
+  }
+
+  public void setCoreAuthentication(CoreAuthenticationProperties coreAuthentication) {
+    this.coreAuthentication = coreAuthentication;
+  }
+
+  private SecurityProperties security;
+
+  @Bean
+  SecurityProperties securityProperties() {
+    return this.getSecurity();
+  }
+
+  /**
+   * Getter for SecurityProperties
+   *
+   * @return Returns the {@link SecurityProperties} object.
+   */
+  public SecurityProperties getSecurity() {
+    return security;
+  }
+
+  /**
+   * Setter for SecurityProperties
+   *
+   * @param security :input {@link SecurityProperties} object
+   */
+  public void setSecurity(SecurityProperties security) {
+    this.security = security;
+  }
+
```

Comment:
cmiiw @woop, but there needs to be `@PostConstruct` `validate()` method in order for validation to work. Frame of reference [Core's FeastProperties](https://github.com/feast-dev/feast/blob/dc159e484766467bca4c14610883461e6be080fa/core/src/main/java/feast/core/config/FeastProperties.java#L231)

Reply:
Validation here would be very useful.